scripts/run-ci-e2e-tests.js

```diff
@@ -70,6 +71,33 @@ try {
 
   const REACT_NATIVE_PACKAGE = path.join(ROOT, 'react-native-*.tgz');
 
+  describe('Set up Verdaccio');
```

Comment:
I would move this block to a `scripts/setup-verdaccio.js` so that other scripts can reuse it.